recommenders/popularity_recommender.py

```diff
@@ -1,26 +1,28 @@
-from recommenders import data_movies, data_ratings
-import pandas as pd
-
-
-def recommend_popular_movies():
-
-    ######################################################################################################
```

Comment:
There are too many unnecessary '#' characters in the comment section. It is recommended to use a single line of '#' to separate sections or a docstring to provide explanations.